### DISCUSSION TITLE
How to NOT marshal a field ?

### DISCUSSION THREAD

**Discussion by aj3423** [pelletier/go-toml#675] [Questions] [answered]
It seems all upper-case fields will be marshal to toml, for example:

```golang
type A struct {
  Name string `toml: any magic to prevent the Name field being marshalled?`
  // other fields...
}
```
I need to access the  `Name` from other package, so it's upper case. But I don't want it to be marshalled, it it possible?

**Comment by moorereason** [comment, marked as answer]
With go-toml/v2 use this:
```go
// Field is ignored
Name string `toml:"-"`
```
This use case is missing from the v2 godocs.

**Reply by pelletier** [reply]
Added this feature to the documentation: https://github.com/pelletier/go-toml/commit/d8997efb5a7b09fe391a1535ad136e8048a02af9.
